tools/ci_build/github/linux/docker/scripts/training/azure_scale_set_vm_mount_test_data.sh

```diff
@@ -1,30 +1,70 @@
 #! /usr/bin/env bash
-if [ -d "/bert_data" ]; then
-sudo umount /bert_data
-fi
-if [ -d "/bert_data" ]; then
-sudo rm -rf /bert_data
-fi
 
-sudo mkdir /bert_data
+function credentialize () {
+    if [ ! -d "/etc/smbcredentials" ]; then
+    sudo mkdir /etc/smbcredentials
+    fi
 
-if [ ! -d "/etc/smbcredentials" ]; then
-sudo mkdir /etc/smbcredentials
-fi
+    if [ -f "/etc/smbcredentials/orttrainingtestdata.cred" ]; then
+    sudo rm /etc/smbcredentials/orttrainingtestdata.cred
+    fi
 
-if [ -f "/etc/smbcredentials/orttrainingtestdata.cred" ]; then
-sudo rm /etc/smbcredentials/orttrainingtestdata.cred
-fi
+    # to create orttrainingtestdata.cred, I have to do: 'sudo bash -c ...'
+    sudo bash -c 'echo "username=orttrainingtestdata" >> /etc/smbcredentials/orttrainingtestdata.cred'
 
-# to create orttrainingtestdata.cred, I have to do: 'sudo bash -c ...'
-sudo bash -c 'echo "username=orttrainingtestdata" >> /etc/smbcredentials/orttrainingtestdata.cred'
+    # $1 get removed (do defend injection attack?) if I do 'sudo bash -c...'
+    # to enable 'sudo echo...' I need to 'sudo chmod 777...' first.
+    sudo chmod 777 /etc/smbcredentials/orttrainingtestdata.cred
+    sudo echo "password=$1" >> /etc/smbcredentials/orttrainingtestdata.cred
 
-# $1 get removed (do defend injection attack?) if I do 'sudo bash -c...'
-# to enable 'sudo echo...' I need to 'sudo chmod 777...' first.
-sudo chmod 777 /etc/smbcredentials/orttrainingtestdata.cred
-sudo echo "password=$1" >> /etc/smbcredentials/orttrainingtestdata.cred
+    sudo chmod 600 /etc/smbcredentials/orttrainingtestdata.cred
+}
 
-sudo chmod 600 /etc/smbcredentials/orttrainingtestdata.cred
+function mount_mnist () {
```

Comment:
I would think that the script takes a source and destination so that there is no further work if we want mount another data.

Reply:
+1 @liqunfu